### DISCUSSION TITLE
"The Whole Truth: A Cosmologist’s Reflections on the Search for Objective Reality"

### DISCUSSION THREAD

**Discussion by RedshiftDrift** [a-cosmology-group/acg#113] [General]
*"The Whole Truth: A Cosmologist’s Reflections on the Search for Objective Reality"*
[https://press.princeton.edu/books/hardcover/9780691231358/the-whole-truth](https://press.princeton.edu/books/hardcover/9780691231358/the-whole-truth)
by P. J. E. Peebles

Has anyone read this book?  On rare occasions Peebles has "slipped" and implied that the Big Bang model may not be the best way to go.  But as a Nobel laureate, he has learned not to openly oppose orthodoxy.

**Comment by ExpEarth** [comment]
Peebles had a similar book "Cosmology's Century" from two years ago, which Craig Fraser was telling me about.  (Craig hasn't made it over to the new ACG yet, but says he will "have a look".)  I think it could be that some of the smarter mainstream cosmologists might want to publish a few of their doubts about the Big Bang, so that if the BB is refuted one day they will be able to claim that they were in there first to challenge it.  I think this new book could be well worth reading.

**Comment by OlliSant** [comment]
It is worth thinking, why the mainstream does not understand that the BB- theory (beginning ans expanding universe)  is impossible, has circular arguments, is against the scientific methods and is against the objective evidence. Why is all that possible? And has continued 60 years. Do we know why this has happened and is possible?

One thing is physics and mathematics. Sometimes is better to ignore them. Cosmology is philosophy too, and they use a wrong philosophy, empiricism and naturalism when these are taken as the only accepted philosophies.

**Reply by khuramonline** [reply]
Yes Natural Philosophy. With Empiricism. No room for Rationalism or Idealism.

And -- Empiricism coupled with Realism --- more precisely -- Epistemological Realism.

That is the way to do all the Theoretical Physics.

Currently the Theoretical Physics is plagued by Idealism and Rationalism with even the symptoms of Animism. And this is exactly how a scientific cult should function.

**Comment by RedshiftDrift** [comment]
In this paper
*"Anomalies in Physical Cosmology"*
[arXiv:2208.05018](https://arxiv.org/abs/2208.05018)
Jim Peebles writes: *"...I argue in this paper that we have empirical evidence that there is a still better theory to be found."*

**Reply by budrap00** [reply]
I didn't find that quote but lots of statements that reduce to this:

`We have instead excellent reason to expect that a better theory to be discovered will look a lot like ΛCDM, because the ΛCDM universe has been shown to look a lot like what is observed.`

That statement is a blatant falsehood. The Cosmos we observe looks nothing like the universe of ΛCDM with its plethora of invisible, undetectable entities and events. The entire paper is just a mathematicist's apologia for the mess that is ΛCDM. Peebles' feeble attempt to wrap himself in a cloak of empiricism is so much posturing. He is a poseur. Peebles is an empiricist the way Stalin was a benevolent leader.  The paper is a fine testament to the intellectual bankruptcy of modern cosmology and its irrational defenders.

**Reply by HanDeBruijn** [reply]
@budrap00 . That quote from Louis is on the first page on the right on top of the paper.
